docs/installation-android.md

```diff
@@ -206,3 +209,20 @@ dependencies {
   compile "com.google.firebase:firebase-perf:11.4.2"
 }
 ```
+
+## 7) Dynamic Links (optional)
```

Comment:
You should reference `https://firebase.google.com/docs/dynamic-links/android/receive`.
Something like:
"Make sure to setup dynamic links for Android as described here..."